install/redis.sh

```diff
@@ -17,7 +17,7 @@ if [ "$install_php" = true ]; then
     sudo phpenmod redis
 fi
 
-sudo pip install redis
+sudo pip3 install redis
```

Comment:
Neither `pip` nor `pip3` should be used with `sudo`.

Reply:
Thanks for the feedback Brian.
I’ve removed the sudo for pip, and from a scan around python/ pip are still used elsewhere so included both versions to install. As the install fails if it cannot find a package, I’ve left a horrible fudge to install python separately so only this one component fails.
I’ve tested on Ubuntu 18.04 and 20.04 and was able to install and then restore from a backup of my emonpi.
I tried setting up a QEMU ARM to test an equivalent to a pi, but haven’t managed to get the networking setup, hence not tested there

Reply:
> I’ve removed the sudo for pip, and from a scan around python/ pip are still used elsewhere so included both versions to install.

Don't understand this statement.

Reply:
Sorry, this relates to your other comments from OEM and relates to my initial removal of pip/ python with only pip3/ python3 from the install step.
After downloading and grepping all of the potential git pulls from the install.sh, there are still a few references to pip/ python, hence I've reworked the install to include both versions, but with a horrible bodge as attempting to run the install on Ubuntu 20 where pip is not available will then stop all the other packages being installed.